app.py

```diff
@@ -180,21 +177,25 @@ def __init__(self, ptree: ParentedTree, senttok: List[str], cgel_terminals = Non
 		if cgel_terminals is not None:
 			self.cgel_tree.update_terminals(cgel_terminals, gaps=True, restore_old_cat=True, restore_old_func=True)
 	
+	# brackettreestr: string representation of ptree in bracket notation, with labels consisting of a POS tag and a function tag separated by a hyphen.
```

Comment:
should be in a triple-quoted string under the method name: https://peps.python.org/pep-0257/#one-line-docstrings

Reply:
Ah, sorry! Will update this right now.